examples/lifecycleDemo.html

```diff
@@ -136,12 +137,13 @@ <h1>Lifecycle Events Log:</h1>
                     // Simulating bootstrap delay of 500ms
                     window.setTimeout(() => {
                         myClientApp.lifecycle.bootstrapped();
-
                         myClientApp.alerting.showToastPopup(
                             lifecycleStatusMessageTitle,
-                            'Bootstrap Complete (500ms delay)', {
-                                id: lifecycleStatusMessageId,
-                                type: 'success'
+                            `Bootstrap Complete (500ms delay):lifecycleStatusMessageId-bootstrap-${count}`, {
+                                id: `lifecycleStatusMessageId-bootstrap-${count++}`,
```

Comment:
looked at the spec and it appears that function arguments are evaluated ltr so these counts should be sync'd.  Has this been the case in your testing?  Even so, I wonder if we should just increment beforehand and not rely on order of function argument evaluation.